CONTRIBUTING.md

```diff
@@ -160,7 +168,7 @@ Before a new component will be reviewed, it must meet the following prerequisite
 If you are creating a new component, you can bootstrap the required file structure with:
 
 ```sh
-yarn createComponent <packageName> <componentName>
+yarn createComponent <packageName> <componentName> <parentName>
```

Comment:
createComponent script now allows you to parentComponent for creation of doc props